src/base-tailwind.css

```
@@ -3,8 +3,11 @@
 
 /* import other libraries' CSS here, if needed */
 @import "@getdokan/dokan-ui/dist/dokan-ui.css";
+@import "../node_modules/@wedevs/plugin-ui/src/components/wordpress/style.css";
+
 
 @source "../src/";
+@source "../node_modules/@wedevs/plugin-ui/dist";
```

Stylelint (17.6.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)